data/monster/lua/#example.lua

```diff
@@ -32,6 +32,18 @@ monster.flags = {
 	staticAttackChance = 70
 }
 
+monster.bestiary = {
+	class = "Dragon",
+	raceId = 39,
+	prowess = 50,
+	expertise = 500,
+	mastery = 1000,
+	charmPoints = 25,
+	stars = 3,
```

Comment:
In the client, this is called "difficulty" and has the levels "harmless", "trivial", "easy", "medium", "hard" and "challenging" (0 to 5 stars)

![image](https://user-images.githubusercontent.com/1993083/232036011-01d11f81-b25c-4a2a-a447-42e5d2ecd02f.png)